keyboards/scarletbandana/scarletbandana.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2017 Cole Markham
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef SCARLETBANDANA_H
+#define SCARLETBANDANA_H
+
+#include "quantum.h"
+
+void reset_keyboard_kb(void);
+
+// This a shortcut to help you visually see your layout.
+// The following is an example using the Planck MIT layout
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
```

Comment:
Could you change this to `LAYOUT`? 

And does this fit an existing layout (eg under /layouts/default/)?

Reply:
```suggestion
#define LAYOUT( \
```